solutions/arrays/rotate-array-clockwise.js

```diff
@@ -0,0 +1,24 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+let arr=readLine().split(" ")

```

Comment:
try to give spaces around binary operators "=", ">" (in line 15) and all the lines where operators are used

Reply:
and also try to use semicolon ";" at the end of each line